server/src/main/java/org/elasticsearch/common/time/DateFormatters.java

```diff
@@ -1374,7 +1373,7 @@ public static DateFormatter forPattern(String input, Locale locale) {
             return YEAR_MONTH;
         } else if ("yearMonthDay".equals(input) || "year_month_day".equals(input)) {
             return YEAR_MONTH_DAY;
-        } else if ("epoch_second".equals(input)) {
+        } else if ("epoch_second".equals(input) || "epoch_seconds".equals(input)) {
```

Comment:
If we are going to add the plural (+1), then we should add a deprecation warning for the non-plural.

Reply:
ok, this requires some more changes in other tests, I will add this in a follow up PR, but do not want to clutter within this PR